platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/service/schemas/V0540RosterBaseSchema.java

```diff
@@ -43,13 +42,19 @@ public V0540RosterBaseSchema() {
     @Override
     public Set<StateDefinition> statesToCreate() {
         return Set.of(
-                StateDefinition.singleton(ROSTER_STATES_KEY, RosterState.PROTOBUF),
-                StateDefinition.onDisk(ROSTER_KEY, ProtoBytes.PROTOBUF, Roster.PROTOBUF, MAX_ROSTERS));
+                StateDefinition.singleton(
+                        RosterStateId.ROSTER_STATE_STATE_ID, RosterStateId.ROSTER_STATE_KEY, RosterState.PROTOBUF),
```

Comment:
Maybe, we should discuss naming conv for these consts, like if we should stick to what's now or:
- `ROSTER_STATE_STATE_ID` -> `ROSTER_STATE_ID`
- `ROSTER_STATE_KEY` -> `ROSTER_STATE_STATE_KEY`

Reply:
The pattern is: <state name>_KEY for state keys and <state name>_STATE_ID for state IDs. Roster service has two states currently: ROSTERS (key/value) and ROSTER_STATE (singleton). It results in 4 constants: ROSTERS_KEY, ROSTERS_STATE_ID, ROSTER_STATE_KEY, and ROSTER_STATE_STATE_ID. It's consistent with the rest of the services